docs/DeveloperGuide.adoc

```diff
@@ -1199,94 +1186,50 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
 
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
-
-|`* * *` |TA |see an overview of events in a week |know what I have that week in a glance
```

Comment:
should keep this user story tho, probably can rephrase it as "upcoming sessions" or smth

Reply:
okie, can